### ISSUE TITLE
Create a method to fill and submit forms for DRYer code

### ISSUE THREAD

**Issue by h3xar0n** [1334/vidbits#4]
Since you have to input the form for every feature you might want to think about creating a file that exports a method to fill the form and submit it where you can pass the values. That way, you wouldn't have to repeat yourself for every feature:

https://github.com/1334/vidbits/blob/b1f93573d14e43fecd617d94ac2f1973e2b6332a/test/features/user-deleting-video-test.js#L8-L14

https://github.com/1334/vidbits/blob/b1f93573d14e43fecd617d94ac2f1973e2b6332a/test/features/user-updating-video-test.js#L9-L15

https://github.com/1334/vidbits/blob/b1f93573d14e43fecd617d94ac2f1973e2b6332a/test/features/user-visiting-create-video-page-test.js#L16-L20

https://github.com/1334/vidbits/blob/b1f93573d14e43fecd617d94ac2f1973e2b6332a/test/features/user-visiting-landing-page-test.js#L17-L21

**Comment by 1334** [comment]
you're right, I've commented this problem on slack with the moderator because I had problems with this approach (which is the one suggested on the exercise steps.
With this approach I'm feature testing the creation of the video on 4 different tests but really only the  `user-visiting-create-video-page-test.js` is concerned with this matter, all other three shouldn't care about how a video is created, they should only care about the precondition that there is a video created on the system.
With this in mind I'd tried to use the `buildVideoObject()` method but I started getting some weird errors and I couldn't make it work.
I don't like this approach at all but I couldn't find a working alternative in time before the deadline.

**Comment by h3xar0n** [comment]
Got it. Would you mind if I make a pull request with one approach later today?

**Comment by 1334** [comment]
that would be awesome, thank you!

**Comment by h3xar0n** [comment]
I think I'm running into the weird errors you mentioned. Going to try another approach tomorrow morning.
